tools/generator/Tests-Core/expected/IntDefinitionAttribute.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+
+namespace Android.Runtime
+{
+	[AttributeUsage (AttributeTargets.Field)]
+	public class IntDefinitionAttribute : Attribute
```

Comment:
I thought adding those fixed the tests? Maybe it was fixed rather by cleaning up files. Anyways they are removed now.